src/jiraIssueStatusUpdates/IssueTransitionConfiguration.gs

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
I don't think so.  Heres my thoughts
1. I think for that purpose I would use the Google caching classes directly so.
2. If the user changes the jira config the cache could be out of date so it could only be cached for a minimal amount of time (say 5 minutes?)
2. The is a performance risk with the number of calls made for transitioning Issues from one state to another but a 5 minute cache would not make an impact. Unfortunately transitioning issues just takes alot of calls. I did consider putting a warning in if the user selects "Status" in the dialog e.g. "This will require more calls to Jira REST API, do not select if the column is not updated" 

If performance becomes an issue *maybe* but by caching you